README.md

```diff
@@ -35,6 +35,8 @@ List of links to the various checkers out there on the web for sites, domains, s
 
 | URL | Description | 💳 |
 | --- | ----------- | -- |
+| https://www.lambdatest.com/ | Test your website in real browsers on multiple devices. | Free trial, free for open source projects, 
```

Comment:
Hi, thanks for this.

As per the [contributor guidelines](https://github.com/Brunty/awesome-checker-services/blob/master/CONTRIBUTING.md), can you ensure any items added into lists are added alphabetically, thanks.